### ISSUE TITLE
EditorCallbacks spews errors

### ISSUE THREAD

**Issue by nasser** [arcadia-unity/Arcadia#251]
Periodically this happens

![image](https://user-images.githubusercontent.com/412966/29989547-b9ac9ff4-8f41-11e7-80c9-2bd818376b0d.png)

```
InvalidCastException: Specified cast is not valid.
arcadia/internal/editor_callbacks$run_repeating_callbacks__81776.invokeStatic (System.Object ) (at <e4b0b027ab0e48aea6acfe2d20a23ec4>:0)
arcadia/internal/editor_callbacks$run_repeating_callbacks__81776.invoke (System.Object ) (at <e4b0b027ab0e48aea6acfe2d20a23ec4>:0)
arcadia/internal/editor_callbacks$run_callbacks__81782.invokeStatic () (at <e4b0b027ab0e48aea6acfe2d20a23ec4>:0)
arcadia/internal/editor_callbacks$run_callbacks__81782.invoke () (at <e4b0b027ab0e48aea6acfe2d20a23ec4>:0)
clojure.lang.Var.invoke () (at <7d6017bd56a24e409770f356e873465b>:0)
Arcadia.EditorCallbacks.RunCallbacks () (at Assets/Arcadia/Editor/EditorCallbacks.cs:43)
UnityEditor.EditorApplication.Internal_CallUpdateFunctions () (at /Users/builduser/buildslave/unity/build/Editor/Mono/EditorApplication.cs:109)
```

**Comment by timsgardner** [comment]
Which commit?

**Comment by nasser** [comment]
`develop`

**Comment by timsgardner** [comment]
may have to do with AOT

**Comment by timsgardner** [comment]
fixed on 3e37564
